src/app/InteractionModelEngine.cpp

```diff
@@ -330,7 +330,7 @@ void InteractionModelEngine::ShutdownMatchingSubscriptions(const Optional<Fabric
 }
 #endif // CHIP_CONFIG_ENABLE_READ_CLIENT
 
-bool InteractionModelEngine::SubjectHasActiveSubscription(const FabricIndex aFabricIndex, const NodeId & subjectID)
+bool InteractionModelEngine::SubjectHasActiveSubscription(const FabricIndex & aFabricIndex, const NodeId & subjectID)
```

Comment:
Not an output argument. Why is this a const ref? FabricIndex is a scalar. It should have just been `FabricIndex aFabricIndex` and `NodeId subjectID`.